### PR TITLE
Add comments about typography principles

### PR DESCRIPTION
**Pull request by huayunh**
Who is going to drop 100 comments in my docit PR when Joe ascends to the manager role...

### Changes proposed in this Pull Request
- Added a few typography principles based on some findings from Yukon.

### Screenshots
image

### To Test
- http://localhost:3000/style/typography

### CODE REVIEW

File: src/docs/style/typography.mdx [hunk 1]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
 
-<React.Fragment>
-    <TypographyTable />
-    <Divider />
-</React.Fragment>
+<TypographyTable />
+
+# Principles
+
+Font variations can create a hierarchy in your information display that guides your user's attention. This involves changing the font's size, weight, and / or color.
```
Comment (joebochill) [inline src/docs/style/typography.mdx:52]:
```suggestion
You should use font variations to establish an information hierarchy in your display that guides users' attention from the most important details to the least important. Creating this information hierarchy involves using multiple font properties, including size, weight, and / or color.
```

File: src/docs/style/typography.mdx [hunk 2]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+
+<ImageGrid
+    images={[TextHierarchy1, TextHierarchy2, TextHierarchy3, TextHierarchy4]}
+    caption={[
+        'You can manipulate many different aspects of your typography to indicate hierarchy.',
```
Comment (joebochill) [inline src/docs/style/typography.mdx:57]:
```suggestion
        'You can manipulate different typography properties to establish hierarchy.',
```

File: src/docs/style/typography.mdx [hunk 3]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+        '1: Font size',
+        '2: Font weight',
+        '3: Font color',
+        '4: Combined',
```
Comment (joebochill) [inline src/docs/style/typography.mdx:61]: Super picky, but the first three are nouns and the fourth is an adjective
```suggestion
        '4: Combination',
```

File: src/docs/style/typography.mdx [hunk 4]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+    ]}
+/>
+
+Before starting on your design, you should understand what contents are important from your user's perspective.
```
Comment (joebochill) [inline src/docs/style/typography.mdx:65]:
```suggestion
As you are crafting your designs, you need to understand which content is the most important from your users' perspective and make that the easiest to find on the page.
```

File: src/docs/style/typography.mdx [hunk 5]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+
+<ImageGrid
+    images={[CompeteAttention1, CompeteAttention2]}
+    caption={[
+        '1: Do not rely solely on font weight changes to indicate an active state. Do not let auxiliary text (the circled numbers on the right) distract your user.',
```
Comment (joebochill) [inline src/docs/style/typography.mdx:70]:
```suggestion
        '1: Do not rely solely on font weight changes to indicate an active state. Avoid having auxiliary information (e.g., the circled numbers on the right) compete for users' attention.',
```

File: src/docs/style/typography.mdx [hunk 6]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+        '2: Use clues like background color and radio buttons to clearly communicate the state.',
+    ]}
+/>
+
+Be careful with font colors too, as they can raise [accessibility issues](/design/accessibility).
+
+<ImageGrid
+    images={[StatusColor1, StatusColor2]}
+    caption={[
+        '1: Be careful when using status color for a large body of text, as they have decreased legibility compared to those with neutral colors.',
```
Comment (joebochill) [inline src/docs/style/typography.mdx:80]:
```suggestion
        '1: Be careful when using status colors for a large body of text, as they have reduced legibility compared to standard text colors.',
```

File: src/docs/style/typography.mdx [hunk 7]
```diff
@@ -20,14 +30,59 @@ PX Blue builds off of the Material Design system, so in general, you may refer t
...
+
+<ImageGrid
+    images={[CompeteAttention1, CompeteAttention2]}
+    caption={[
+        '1: Do not rely solely on font weight changes to indicate an active state. Do not let auxiliary text (the circled numbers on the right) distract your user.',
+        '2: Use clues like background color and radio buttons to clearly communicate the state.',
+    ]}
+/>
+
+Be careful with font colors too, as they can raise [accessibility issues](/design/accessibility).
```
Comment (joebochill) [inline src/docs/style/typography.mdx:75]:
```suggestion
When changing font colors, make sure to take into consideration the [accessibility](/design/accessibility) of your chosen colors.
```

**Review by joebochill — COMMENTED** [review]
I'm not sure how I feel about the radio buttons here...I don't know that I've seen that anywhere before for sidebar filtering:
image

**Comment by huayunh** [comment]
> I'm not sure how I feel about the radio buttons here...I don't know that I've seen that anywhere before for sidebar filtering:

what's the part that makes you feel off?

**Comment by joebochill** [comment]
> > I'm not sure how I feel about the radio buttons here...I don't know that I've seen that anywhere before for sidebar filtering:
> 
> what's the part that makes you feel off?

The radio button itself mostly. I feel like whenever I see a sidebar filter like that, 9 times out of 10 it's either just an item that you click (if you can select only one), or checkboxes for multiselect:
image


I did find an example that uses radios, and it doesn't bother me that much. 
image

Idk, maybe our radio button just feels really big for some reason. Or maybe there's just too much blue happening.

**Comment by huayunh** [comment]
Hmm... I oughta be concerned. How do you like this screen?

image

**Comment by huayunh** [comment]
_(I reserve my opinion about Amazon's UI design in a public space like this)_

**Comment by joebochill** [comment]
The more I look at it the more I think the radio buttons are fine. They don't really bother me that muhc, I just wasn't used to seeing it.

**Review by joebochill — APPROVED** [review]
Looks good

**Pull request merged**